src/model/test/Space_GTest.cpp

```diff
@@ -2038,10 +2038,13 @@ TEST_F(ModelFixture, Space_intersectSurfaces_degenerate3) {
 /// First of all we need to remove surfaces that overlap within the same space
 /// Second of all we use a different removeSpikes method that shrinks and expands the polygon
 TEST_F(ModelFixture, RemoveSpikesAndOverlaps) {
-  osversion::VersionTranslator translator;
-  //model::OptionalModel model = translator.loadModel(toPath("./whole_bldg_partially_matched.osm"));
-  //boost::optional<Model> model = Model::load(toPath("./7-7 Windows Complete.osm"));
-  boost::optional<Model> model = Model::load(toPath("./two_stories_pre_intersect.osm"));
+
+  openstudio::path path = resourcesPath() / toPath("model/two_stories_pre_intersect.osm");
+  // openstudio::path path = resourcesPath() / toPath("model/whole_bldg_partially_matched.osm");
+  // openstudio::path path = resourcesPath() / toPath("model/7-7 Windows Complete.osm");
```

Comment:
If you need to test all three, you can make a free function in the test file, with their EXPECT/ASSERT statements, and pass the OSM as argument.